core-model/src/main/java/com/sequenceiq/cloudbreak/domain/security/User.java

```diff
@@ -0,0 +1,143 @@
+package com.sequenceiq.cloudbreak.domain.security;
+
+import java.util.Set;
+
+import javax.persistence.Basic;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import com.sequenceiq.cloudbreak.domain.ProvisionEntity;
+import com.sequenceiq.cloudbreak.domain.json.Json;
+import com.sequenceiq.cloudbreak.domain.json.JsonStringSetUtils;
+import com.sequenceiq.cloudbreak.domain.json.JsonToString;
+
+@Entity
+@Table(name = "users")
+public class User implements ProvisionEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "users_generator")
+    @SequenceGenerator(name = "users_generator", sequenceName = "users_id_seq", allocationSize = 1)
+    private Long id;
+
+    private String name;
+
+    @Basic(optional = false)
+    private String email;
+
+    private String company;
+
+    private String idp;
```

Comment:
We don't know what to write into this field, I think it's better if we delete it for now